### PR TITLE
Debian Updates

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ola (0.9.6-1) unstable; urgency=medium
```
Comment (peternewman) [inline debian/changelog]: I guess in theory this should be urgency low still.

File: debian/control [hunk 2]
```diff
@@ -1,43 +1,63 @@
...
 Architecture: any
 Depends: ola (= ${binary:Version}), ${misc:Depends}
-Description: Open Lighting Architecture
+Description: Open Lighting Architecture - development libraries
+ The DMX512 standard for Digital MultipleX is used for digital
+ communication networks commonly used to control stage lighting and
+ effects.
+ .
  The Open Lighting Architecture (OLA) provides a plugin framework for
  distributing DMX512 control signals on Mac and Linux.
```
Comment (peternewman) [inline debian/control]: We should probably add Windows now, plus *BSD?

File: debian/control [hunk 3]
```diff
@@ -1,43 +1,63 @@
...
+Description: Open Lighting Architecture - Python Classes
+ The DMX512 standard for Digital MultipleX is used for digital
+ communication networks commonly used to control stage lighting and
+ effects.
+ .
  The Open Lighting Architecture (OLA) provides a plugin framework for
  distributing DMX512 control signals on Mac and Linux.
  .
- OLA is written and maintained by Simon Newton <nomis52@gmail.com>
+ This package contains the files required for plugins or to control the OLA
```
Comment (peternewman) [inline debian/control]: No plugins here.

File: debian/control (hunk 2 above)
Comment (nomis52) [inline debian/control]: Removed the platform bits, since this isn't relevant for the debian packages.

**Pull request merged**